docsite/rst/playbooks_best_practices.rst

```diff
@@ -27,8 +27,8 @@ Directory Layout
 
 The top level of the directory would contain files and directories like so::
 
-    production                # inventory file for production servers
-    staging                   # inventory file for staging environment 
+    production                # inventory file for production servers (or directory that contains hosts.yml)
+    staging                   # inventory file for staging environment (or directory that contains hosts.yml)
```

Comment:
I'm still unclear about switching from production to staging easily. I just propose it because it seems coded in this way for hosts.yml file. I don't understand yet the rules for group_vars...
Anyway, that is fine to cancel this part of my PR. Maybe a playbook command line sample could make it more clear for everybody ?
Cheers,